module/proto/gravity/v1/msgs.proto

```diff
@@ -1,237 +1,176 @@
-  }
-  rpc ConfirmLogicCall(MsgConfirmLogicCall) returns (MsgConfirmLogicCallResponse) {
-    option (google.api.http).post = "/gravity/v1/confim_logic";
-  }
-  rpc DepositClaim(MsgDepositClaim) returns (MsgDepositClaimResponse) {
-    option (google.api.http).post = "/gravity/v1/deposit_claim";
+  rpc Transfer(MsgTransfer) returns (MsgTransferResponse) {
+    option (google.api.http).post = "/gravity/v1/transfer";
   }
-  rpc WithdrawClaim(MsgWithdrawClaim) returns (MsgWithdrawClaimResponse) {
-    option (google.api.http).post = "/gravity/v1/withdraw_claim";
+  rpc CancelTransfer(MsgCancelTransfer) returns (MsgCancelTransferResponse) {
+    option (google.api.http).post = "/peggy/v1/transfer/cancel";
   }
-  rpc ERC20DeployedClaim(MsgERC20DeployedClaim)
-      returns (MsgERC20DeployedClaimResponse) {
-    option (google.api.http).post = "/gravity/v1/erc20_deployed_claim";
+  rpc RequestBatchTx(MsgRequestBatchTx) returns (MsgRequestBatchTxResponse) {
+    option (google.api.http).post = "/gravity/v1/batchtx/request";
   }
-  rpc LogicCallExecutedClaim(MsgLogicCallExecutedClaim)
-      returns (MsgLogicCallExecutedClaimResponse) {
-    option (google.api.http).post = "/gravity/v1/logic_call_executed_claim";
+  rpc SubmitEthereumSignature(MsgSubmitEthereumSignature) returns (MsgSubmitEthereumSignatureResponse) {
+    option (google.api.http).post = "/peggy/v1/ethereum_signature";
   }
-  rpc SetOrchestratorAddress(MsgSetOrchestratorAddress) returns (MsgSetOrchestratorAddressResponse) {
-    option (google.api.http).post = "/gravity/v1/set_orchestrator_address";
+  rpc SubmitEthereumEvent(MsgSubmitEthereumEvent) returns (MsgSubmitEthereumEventResponse) {
+    option (google.api.http).post = "/peggy/v1/ethereum_event";
   }
-  rpc CancelSendToEth(MsgCancelSendToEth) returns (MsgCancelSendToEthResponse) {
-    option (google.api.http).post = "/gravity/v1/cancel_send_to_eth";
+  rpc SetDelegateKeys(MsgDelegateKeys) returns (MsgDelegateKeysResponse) {
+    option (google.api.http).post = "/peggy/v1/delegate_keys";
   }
 }
 
-// MsgSetOrchestratorAddress
-// this message allows validators to delegate their voting responsibilities
-// to a given key. This key is then used as an optional authentication method
-// for sigining oracle claims
-// VALIDATOR
-// The validator field is a cosmosvaloper1... string (i.e. sdk.ValAddress)
-// that references a validator in the active set
-// ORCHESTRATOR
-// The orchestrator field is a cosmos1... string  (i.e. sdk.AccAddress) that
-// references the key that is being delegated to
-// ETH_ADDRESS
-// This is a hex encoded 0x Ethereum public key that will be used by this validator
-// on Ethereum
-message MsgSetOrchestratorAddress {
-  string validator    = 1;
-  string orchestrator = 2;
-  string eth_address  = 3;
+// MsgTransfer submits a transfer attempt to bridge an asset over to Ethereum.
+// The transfer will be stored and then included in a batch and then
+// submitted to Ethereum.
+message MsgTransfer {
```

Comment:
I liked the old naming better since it was more descriptive. `Transfer` is a very bland, generic word and imparts no information. `SendToEthereum` would be much clearer.

Reply:
After thinking about this I tend to agree. Will change.